03_letters/starfallz/main_test.go

```diff
@@ -0,0 +1,103 @@
+package main
+
+import (
+	"bytes"
+	"reflect"
+	"strconv"
+	"testing"
+)
+
+func TestMainFunction(t *testing.T) {
+	t.Run("Test main to return string values of letters(aba) with proper formatting", func(t *testing.T) {
+		var buf bytes.Buffer
+		out = &buf
+
+		main()
+
+		expected := strconv.Quote(`a:2
+b:1`)
+		actual := strconv.Quote(buf.String())
+
+		if expected != actual {
+			t.Errorf("Unexpected output, expected: %s, actual: %s", expected, actual)
+		}
+	})
```

Comment:
I'm not usually very big on getting things formatted properly but it definitely looks weird with the long line plus the extra spacing plus the line break for the literals. 

Also, don't capitalize error strings throughout (from https://github.com/anz-bank/go-course/wiki/Lab-1:-Collected-Code-Review-Comments, see also https://github.com/golang/go/wiki/CodeReviewComments#error-strings